local-cli/server/runServer.js

```diff
@@ -84,6 +84,9 @@ function getPackagerServer(args, config) {
     args.transformer ? path.resolve(args.transformer) :
     typeof config.getTransformModulePath === 'function' ? config.getTransformModulePath() :
     undefined;
+  const polyfillModuleNames =
+    typeof config.getPolyfillModuleNames === 'function' ? config.getPolyfillModuleNames() :
```

Comment:
Could you make it non-nullable instead? That's the new pattern we tend to stick with for new fields, by providing a default value that is an empty function in the `defaultConfig` object. You can take example on the way we added the new configuration function `getSourceExts`: https://github.com/eacaps/react-native/commit/4a86f93982ede66a993a72e31cadb85d9bd4b3a3
(without going as far as to add it as a command line argument, that I don't think is necessary in that case)